src/tracks/ui/TrackView.cpp

```diff
@@ -153,6 +153,21 @@ void TrackView::DoSetY(int y)
    mY = y;
 }
 
+int TrackView::GetTop() const
+{
+    return mY;
+}
+
+int TrackView::GetBottom() const
+{
+    return GetTop() + GetHeight();
```

Comment:
Is this an off-by-one error?  Should you subtract 1?

See how wxRect::GetBottom() is defined in gdicmn.h

Or should you rename the function to express a different purpose?

Reply:
Probably you are right